internal/mongodoc/doc.go

```diff
@@ -182,10 +185,14 @@ type BaseEntity struct {
 	// The permissions apply to all revisions.
 	ACLs ACL
```

Comment:
We could mentions that ACLs apply to unpublished entities, not assigned a stable or development channel.